magento/magento2
# Add trim filter to first, middle and lastname.

### Description
The additional trim form field data filter was created and added as
input filter to the firstname, lastname and middlename fields
of the customer_address and customer entities.

### Fixed Issues (if relevant)

1. magento/magento2#10415: Customer First and Last names not being trimmed of leading and trailing spaces on save.

### Manual testing scenarios

1. Create a new customer from the Customers > All Customers > Create New Customer window
2. Create First name: "John " Last name: " Doe ", add a fake email address.
3. Click save
4. Search and edit the contact "John Doe "
5. Take note that the spaces are no longer in the users first, middle and last names. Same goes for the customer names set on the customer addresses.

### Contribution checklist
